styleguide/git/commit-messages.md

```diff
@@ -0,0 +1,51 @@
+# Commit Messages
+
+The commit message must follow the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
+
+As we build automated Change Logs for some projects please make sure you only use the specified types and formats.
```

Comment:
Well we do not have automated releases yet. But I will add what will be part of the release notes generated.

Reply:
@rcky As I was checking what it would do with code I figured out, that chore isn't actually a part of the angular guidelines, so i rewrote the readme. please have a look. I added a note to look at the documentation of the program generating the versions.
We are using the following on in some of our projects. IMO it generates at least a bugfix version dump. But if there is only a chore the change log will be empty.
https://gitlab.com/hyper-expanse/open-source/semantic-release-github#readme